src/ingestion/serializers.py

```
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+from .models import TicketEvent
+
+class TicketEventSerializer(serializers.ModelSerializer):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)